src/Symfony/Component/Yaml/Parser.php

```diff
@@ -200,7 +200,7 @@ private function doParse(string $value, int $flags)
                     array_pop($this->refsBeingParsed);
                 }
             } elseif (
-                self::preg_match('#^(?P<key>(?:![^\s]++\s++)?(?:'.Inline::REGEX_QUOTED_STRING.'|(?:!?!php/const:)?[^ \'"\[\{!].*?)) *\:( ++(?P<value>.+))?$#u', rtrim($this->currentLine), $values)
+                self::preg_match('#^(?P<key>(?:![^\s]++\s++)?(?:'.Inline::REGEX_QUOTED_STRING.'|(?:!?!php/const:)?[^ \'"\[\{!].*?)) *\:(( |\t)++(?P<value>.+))?$#u', rtrim($this->currentLine), $values)
```

Comment:
Why group (`( |\t)`) insteand of list (`[ \t]`)? Lists are a bit faster.

Reply:
No specific reason except that I didn't know that there was a performance difference. Is it significant though?
I have no strong opinion about one or the other, but this PR is merged so it'd require a new PR.

Reply:
@bertramakers I did a benchmark, and list is 1% faster. I do not know if it is considered a micro-optimization, but since it's a parser, maybe it's something significant.

Reply:
I have no idea either, I just discovered this bug while using the Yaml component in my own app and reported and fixed it to the best of my ability 😄 I think you can always open another PR to optimize it if you think it's worth it!

Reply:
@bertramakers yeah, no problem! The fix works.